mtui/commands/reloadoqa.py

```diff
@@ -1,7 +1,7 @@
 from logging import getLogger
 
 from ..connector.openqa import AutoOpenQA, KernelOpenQA
-from ..utils import complete_choices, requires_update
+from ..utils import complete_choices, requires_update  # noqa: F401
```

Comment:
#WaroyaltyCreations
If your looking for Avant and or Acentric music for your playlist.
Subscribe to my YouTube channel @JingleManning /WaroyaltyCreations .
For information, keep eyes out for new projects.